data/api/server-server/definitions/event-schemas/m.receipt.yaml

```diff
@@ -42,40 +42,42 @@ allOf:
             "m.read":
               type: object
               description: Read receipts for users in the room.
-              title: User Read Receipt
-              properties:
-                event_ids:
-                  type: array
-                  description: |-
-                    The extremity event IDs that the user has read up to.
-                  minItems: 1
-                  maxItems: 1
-                  items:
-                    type: string
-                  example: ['$read_this_event:matrix.org']
-                data:
-                  type: object
-                  description: Metadata for the read receipt.
-                  title: Read Receipt Metadata
-                  properties:
-                    ts:
-                      type: integer
-                      format: int64
-                      description: |-
-                        A POSIX timestamp in milliseconds for when the user read
-                        the event specified in the read receipt.
-                      example: 1533358089009
-                    thread_id:
-                        type: string
-                        x-addedInMatrixVersion: "1.4"
+              additionalProperties:
+                type: object
```

Comment:
I think we should document what the key of the object is (the user id), much as we do for the `content` object above it: https://github.com/matrix-org/matrix-spec/blob/9c995dfcb6520d40372e7f5ee7004de001aefc7d/data/api/server-server/definitions/event-schemas/m.receipt.yaml#L34-L35